backend/src/modules/AddComment/service/AddComment.service.js

```diff
@@ -0,0 +1,19 @@
+import { Comment } from './AddComment.js';
+
+export class AddCommentService {
+  constructor(repository) {
+    this.repository = repository;
+  }
+
+  async addComment(comment) {
+    const commentData = new Comment({
```

Comment:
Nowy komentarz tworzysz już w kontrolerze, tutaj robisz to drugi raz (wydaje mi się że) niepotrzebnie. 
Brakuje też w serwisie walidacji. 
Zamiast tworzyć nowy obiekt powinieneś sprawdzić, czy to co zostało tu przekazane jest zgodne ze schematem, przy pomocy biblioteki joi. W przeciwnym razie wyrzucić błąd.

Reply:
Możliwe, dlatego napisałam że wydaje mi się :)
Odnośnie joi tak, ale w pozostałych ścieżkach używamy tej biblioteki, więc fajnie by było utrzymać jakąś spójność (mimo, że osobne moduły). 

Z naszych ustaleń zrozumiałam, że walidacja odbywa się w serwisie. W serwisie powinniśmy zapewnić, że to co przekazujesz do repozytorium jest poprawne. Obiekt domenowy wykorzystujemy już w repozytorium i do danych **zwracanych** zwracanych z repo. Popraw mnie jeśli się mylę :)

Reply:
Tylko jako "Service" rozumiesz katalog "service" (wtedy tak), czy plik "SthService.js" (wtedy niekoniecznie)? 
Tzn, jeśli zrobisz walidację w "SthService.js" nie bedzie "źle", ale o wiele lepiej bedzie zrobić to w obiekcie domenowym - wtedy mamy 100% pewność, że dane są prawidłowe w obiekcie domenowym :)